release/release.go

```diff
@@ -117,3 +124,10 @@ type GcpImage struct {
 	Family  string `json:"family,omitempty"`
 	Name    string `json:"name,omitempty"`
 }
+
+// IBMCloudObject represents an IBMCloud/PowerVS cloud object - which is a cos bucket image
```

Comment:
Nit: if `cos` is meant to mean `CoreOS`, we don't actually use that abbreviation.

Reply:
it means `cloud-object-storage` which is specific to the IBM Cloud storage. i will call this out.